src/main/ank/1.5-products-and-coproducts.md

```diff
@@ -0,0 +1,275 @@
+```Haskell
+absurd :: Void -> a
+```
+```kotlin
+fun <A> absurd(): (Nothing) -> A
+```
+................
+```Haskell
+unit :: a -> ()
+unit _ = ()
+```
+```kotlin:ank:silent
+val unit: Unit = Unit
+```
+................
+```Haskell
+yes :: a -> Bool
+yes _ = True
+```
+```kotlin:ank:silent
+val yes: Boolean = true
+```
+................
+```Haskell
+no :: a -> Bool
+no _ = False
+```
+```kotlin:ank:silent
+val no: Boolean = false
```

Comment:
Same

Reply:
I can make them fun if it's more clear

Reply:
I can make them fun if it's more clear

Reply:
@nomisRev is this closer to what you envisioned or do you mean using `val` with `(A) -> B` return types?

Reply:
Exactly what I was thinking of.